chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# stop on first error
+set -e
```

Comment:
We also use `set -eo pipefail` - https://gist.github.com/vncsna/64825d5609c146e80de8b1fd623011ca